state_synchronizer/src/coordinator.rs

```diff
@@ -227,12 +224,13 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
         let is_update = version > self.known_version;
         self.known_version = std::cmp::max(version, self.known_version);
         if is_update {
-            if let Ok(duration_since_last_commit) =
-                SystemTime::now().duration_since(self.last_commit.unwrap_or(UNIX_EPOCH))
+            if let Some(last_request_tst) =
+                self.peer_manager.get_request_time(self.known_version + 1)
             {
-                counters::SYNC_PROGRESS_DURATION.observe_duration(duration_since_last_commit);
+                if let Ok(duration) = SystemTime::now().duration_since(last_request_tst) {
```

Comment:
It seems that this is changing the value of the SYNC_PROGRESS_DURATION counter.
In the previous code we would measure duration since last commit, while in the new version we measure duration since last sent request, which is updated upon every timeout.
Do I read it right?

Reply:
yes, which measures approximately the same on happy path
it doesn't take into account time spent for interactions with byzantine peers
don't want to keep `last_commit_time` internal variable just for that counter

Reply:
Yea, I guess we'd still want to have some understanding of time to retrieve the chunk (across all the retries etc.), but we already have the counter for the number of timeouts, so it is somewhat representative.
@bors-libra r+